app/models/ContributedToDuringRemedyPeriod.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import play.api.i18n.Messages
+import uk.gov.hmrc.govukfrontend.views.viewmodels.checkboxes.CheckboxItem
+import uk.gov.hmrc.govukfrontend.views.viewmodels.content.Text
+import viewmodels.govuk.checkbox._
+
+sealed trait ContributedToDuringRemedyPeriod
+
+object ContributedToDuringRemedyPeriod extends Enumerable.Implicits {
+
+  case object Definedcontribution extends WithName("definedContribution") with ContributedToDuringRemedyPeriod
+  case object Definedbenefit extends WithName("definedBenefit") with ContributedToDuringRemedyPeriod
```

Comment:
The casing of the case objects here is unexpected. Can it be DefinedContribution + DefinedBenefit?

Reply:
Corrected